packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -306,6 +306,22 @@ export const formatters: Formatters = {
     format: ({ user, mfa_device_name, mfa_device_type }) =>
       `User [${user}] deleted ${mfa_device_type} device [${mfa_device_name}]`,
   },
+  [CodeEnum.BILLING_CARD_CREATE]: {
+    desc: 'Billing Credit Card Created',
+    format: ({ user }) => `A billing credit card has been created by ${user}`,
```

Comment:
@kimlisa 
Let's be consistent with other events 
-> `User [${user}] has updated a credit card`

Reply:
btw, feel free to pick an icon to use for billing events.

Reply:
revised
![image](https://user-images.githubusercontent.com/43280172/111561951-9150ed00-8752-11eb-8eeb-7c627ad92482.png)

Reply:
Lets change the event description to:

"Credit Card Created"
"Credit Card Updated"
"Credit Card Deleted"
"Billing Information Updated"

Reply:
Billing Account -> Billing Information

Reply:
revised
![image](https://user-images.githubusercontent.com/43280172/111563048-767f7800-8754-11eb-9fb8-345eef1cc244.png)